sentry-samples/sentry-samples-jul/src/main/java/io/sentry/samples/jul/Main.java

```diff
@@ -23,5 +24,7 @@ public static void main(String[] args) {
     } catch (Exception e) {
       LOGGER.log(Level.SEVERE, "Something went wrong", e);
     }
+    // it didn't help either, looks like SDK is not even initializing.
```

Comment:
This sample has to be started with `../../gradlew run` as there must be a system property `Djava.util.logging.config.file` pointing to `logging.properties` file set.

Reply:
@maciejwalkowiak yep, its on the readme, I've used this command